hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/BackpressureRegulator.java

```diff
@@ -6,8 +6,9 @@
 import com.hazelcast.spi.Operation;
 
 import java.util.Random;
-import java.util.concurrent.atomic.AtomicInteger;
 
+import static com.hazelcast.client.impl.protocol.util.BitUtil.CACHE_LINE_LENGTH;
```

Comment:
I already removed the dependency. 

Can you make sure that after your cleanup, there is no duplication?